Documentation/Signoff/Wireless Power Transfer.md

```diff
@@ -0,0 +1,149 @@
+|4.2             |0.5                    |0.15              |2.1                |1.35                 |10          |15.55           |
+
+
+
+## B.O.M.
+
+|Item                                 |Location       |Quantity |Price      |Total Price |
+|-------------------------------------|---------------|---------|-----------|------------|
+|AA Batteries                         |Amazon         |4+       |$16.43     |$16.43      |
+|4xAA Battery Pack                    |Adafruit       |1        |$2.95      |$2.95       |
+|68uH Inductor                        |Digikey        |2        |$1.17      |$2.34       |
+|150nF Capacitor                      |Digikey        |1        |$0.44      |$0.44       |
+|100nF Capacitor                      |Digikey        |1        |$0.40      |$0.40       |
+|33nF Capacitor                       |Digikey        |1        |$0.31      |$0.31       |
+|Transmitter 4.95uH Coil              |Digikey        |1        |$7.44      |$7.44       |
+|100ohm Resistor                      |Digikey        |2        |$0.10      |$0.20       |
+|10nF Capacitor                       |Digikey        |2        |$0.10      |$0.20       |
+|Transmitter Mosfet                   |Digikey        |2        |$1.70      |$3.40       |
+|SK34SMA-3G   Diode                   |Digikey        |5        |$0.40      |$2.00       |
+|BZX84C16-7-F Diode                   |Digikey        |2        |$0.15      |$0.30       |
+|Receiver 46uH Coil                   |Digikey        |1        |$4.03      |$4.03       |
+|4.7nF Capacitor                      |Digikey        |2        |$0.43      |$0.86       |
+|22nF Capacitor                       |Digikey        |2        |$0.88      |$1.76       |
+|1.5nF Capacitor                      |Digikey        |1        |$1.08      |$1.08       |
+|10uF Capacitor                       |Digikey        |1        |$0.30      |$0.30       |
+|470kohm Resistor                     |Digikey        |2        |$0.10      |$0.20       |
+|2.2uF Capacitor                      |Digikey        |1        |$0.17      |$0.17       |
+|15uH Inductor                        |Digikey        |1        |$1.19      |$1.19       |
+|3.01kohm Resistor                    |Digikey        |1        |$0.10      |$0.10       |
+|1Mohm Resistor                       |Digikey        |1        |$0.10      |$0.10       |
+|10kohm Resistor                      |Digikey        |1        |$0.10      |$0.10       |
+|1.35Mohm Resistor                    |Digikey        |1        |$0.66      |$0.66       |
+|22uF Capacitor                       |Digikey        |1        |$0.33      |$0.33       |
+|9V Lithium-ion Battery               |Amazon         |1        |$18.99     |$18.99      |
+|9V Lithium-ion Battery Connector     |Amazon         |1        |$4.99   v  |$4.99       |
+|                                     |               |         |Total:     |$71.27      |
```

Comment:
Where are part numbers? Just specifying the component rating does not fully specify the component.